doc/dev/cephadm/developing-cephadm.rst

```diff
@@ -102,3 +102,126 @@ another handler is executing.
 This means we should do very few synchronous calls to remote hosts.
 As a guideline, cephadm should do at most ``O(1)`` network calls in CLI handlers.
 Everything else should be done asynchronously in other threads, like ``serve()``.
+
+Kcli: a virtualization management tool to make easy orchestrators development
```

Comment:
Can you add this section before `Note regarding network calls from CLI handlers` please? In this chapter we have:

1. Development evironmnets
2. Note regarding network calls from CLI handlers

would make sense to have them grouped properly